WoWsShipBuilder.Web/Components/ShipStatsComponent.razor

```diff
@@ -139,14 +140,26 @@
                                 <!--Border bottom initial needed to fix the first expander having a wrong color when closed-->
                                 <MudExpansionPanel DisableGutters Text="@Localizer.GetGameLocalization(torpedo.Name).Localization" IsInitiallyExpanded="@(AppSettings.WebAppSettings?.OpenAllAmmoExpandersByDefault ?? true)" Style="border-bottom: initial" Class="child-expander rounded-0">
```

Comment:
This also needs the key property, the same applies for the data fragments. This is just a reminder for me to fix this, ignore it for this PR